documentation/asciidoc/accessories/iqaudio/faq.adoc

```diff
@@ -1,13 +1,12 @@
-=== Saving alsamixer settings
+=== Saving AlsaMixer settings
 
-Storing the alsamixer setting can be achieved by performing the following at the command line:
+To store the AlsaMixer setting, add the following at the command line:
```

Comment:
setting -> settings? (to match the heading-text)

Reply:
Fixed